tests/sensitivities/test_create_design.py

```diff
@@ -161,6 +164,24 @@ def test_generate_onebyone(tmpdir):
 
     # MULTZ_ILE contains random numbers so we won't test it here.
 
+    diskmetadata = pd.read_excel(
+        "designmatrix.xlsx", sheet_name="Metadata", engine="openpyxl"
+    )
+
+    assert (diskmetadata.columns == ["Description", "Value"]).all()
+    assert diskmetadata["Description"].iloc[0] == "Created using fmu-tools version:"
+    assert diskmetadata["Value"].iloc[0] == fmu.tools.__version__
+    assert diskmetadata["Description"].iloc[1] == "Created on:"
+
+    # For the timestamp, we can't check the exact value since it will differ
+    # each time the test runs. Instead, we can verify it's a valid datetime string
+    # by attempting to parse it
+    timestamp = diskmetadata["Value"].iloc[1]
+    try:
```

Comment:
assert instead that `datetime.fromisoformat()` does not err

Reply:
You could mock datetime.now() if you want something exact to assert, but I am fine with this :)

Reply:
Thanks, I'm now using `datetime.fromisoformat()` instead. Don't want to do mocking though! 😬